ts/genesis.ts

```diff
@@ -35,17 +41,24 @@ export class Genesis {
         public readonly auxiliary: Auxiliary
     ) {}
 
-    static fromConfig(path: string) {
-        const genesis = fs.readFileSync(path).toString();
-        const { parameters, addresses, auxiliary } = JSON.parse(genesis);
+    public static async fromConfig(
+        path: string = "./genesis.json"
+    ): Promise<Genesis> {
+        const genesisPath = resolve(path);
+        console.info(`loading genesis from ${genesisPath}`);
+        const genesisJSONStr = await readFileAsync(genesisPath, {
```

Comment:
non-blocking NodeJS i/o APIs are preferred over `somethingSync` so we don't block the event loop.